robottelo/hosts.py

```diff
@@ -1023,12 +1023,14 @@ def configure_puppet(self, proxy_hostname=None):
         self.execute('/opt/puppetlabs/bin/puppet agent -t')
```

Comment:
Nowadays (since Puppet 6) if you want to bootstrap you use `puppet ssl bootstrap`. Then you can sign it. You may want to pass `--waitforcert 0` as well to prevent it waiting for the server to sign it.